### PR TITLE
Plot hotspots on map for consensus transaction detail page

### LINKED ISSUE #92: Draw Map of Hotspots in Consensus
A request from Andrew T - 
visualize consensus membership by plotting hotspot locations on a map.

example consensus group:
https://explorer.helium.com/txns/vCaP6J1b90dnXP3WrrOfdzD-IDqvUEgbZSrlb1XbpZk

### PR DESCRIPTION
**Pull request by danielcolinjames**
Fixes #92 

NOTE: this should get merged after #107 — I brought that work into this PR because I had to edit some of the same data fetching code for the [txnid].js page

This PR adds a custom transaction detail view for transactions of the type `consensus_group_v1` so that we can display the data from a consensus transaction more nicely, e.g. plotting all the consensus members on a map (instead of using the <Fallback /> transaction type and just displaying the raw JSON of the transaction data)

I also made a helper function that takes in an array of lat/lons and finds the max & min lat/lon to give a bounding box, so Mapbox can zoom and pan appropriately to any set of consensus members. E.g. if the min was in California and the max was in Iceland, for example, it would zoom out much further, and would only zoom into North America if the min was in California and the max in New York.

Preview of how it looks:

### CODE REVIEW

**Comment by danielcolinjames** [comment]
Example pages to try it on:

https://helium-explorer-pr-101.herokuapp.com/txns/bYLJnCHL2bH9eqhipwOE0EjJql_KV_AAJizlBdKQFO4
https://helium-explorer-pr-101.herokuapp.com/txns/Ux-TUk_tvdE9oMWbR4M-dPlmBRXBUQnxk1m2xeLk-sw
https://helium-explorer-pr-101.herokuapp.com/txns/I1SZfvT4hsPwM2gKJeHHMSqhfaDfxeHg5uVis7s1LnY
https://helium-explorer-pr-101.herokuapp.com/txns/ULC2bnp9mu3xcBqj5hCsVRHCZs31CkDc27MmUbCg4kk
https://helium-explorer-pr-101.herokuapp.com/txns/foJOoLb5EjPfNDlKLI2U5TYoltl3oR95gLT1hH6tELc
https://helium-explorer-pr-101.herokuapp.com/txns/Mh-7fvs2RqgyaAQmLpJTTTaZ74dM1sLrULlMdsU2DGE
https://helium-explorer-pr-101.herokuapp.com/txns/vCaP6J1b90dnXP3WrrOfdzD-IDqvUEgbZSrlb1XbpZk

File: components/Txns/PocReceiptsV1.js [hunk 1]
```diff
@@ -42,64 +42,151 @@ const PoCTableHeader = ({ tooltipText, title }) => {
...
+                                p.receipt.snr
+                                  ? `${p.receipt?.snr.toFixed(2)}dB`
+                                  : ''
+                              }${
+                                p.receipt !== null &&
+                                p.receipt.datarate !== undefined
+                                  ? Array.isArray(p.receipt.datarate)
+                                    ? `${
+                                        p.receipt.datarate.length > 0
+                                          ? `, ${String.fromCharCode.apply(
```
Comment (allenan) [inline components/Txns/PocReceiptsV1.js:158]: nested ternary logic can be confusing to interpret. could this be extracted to a little helper function?

File: pages/accounts/[accountid].js [hunk 2]
```diff
@@ -25,7 +25,31 @@ const AccountAddress = ({ address, truncate = false }) => {
...
   return (
-    <AppLayout>
+    <AppLayout
+      title={`${account.address.substring(0, 5)}... | Account`}
+      description={`An account on the Helium blockchain with ${account.balance.floatBalance.toLocaleString(
+        undefined,
+        { minimumFractionDigits: 2, maximumFractionDigits: 2 },
+      )} HNT${
+        account.dcBalance.floatBalance > 0
+          ? `, ${account.dcBalance.floatBalance.toLocaleString(undefined, {
```
Comment (allenan) [inline pages/accounts/[accountid].js:35]: can we use `toString` here rather than `floatBalance`?

File: components/Txns/PocReceiptsV1.js [hunk 3]
```diff
@@ -42,64 +42,151 @@ const PoCTableHeader = ({ tooltipText, title }) => {
...
+                            {idx !== 0 && (
+                              <span
+                                className={`poc-path-line-above ${
+                                  txn.path[idx - 1].receipt &&
+                                  txn.path[idx - 1].receipt.origin === 'radio'
+                                    ? 'poc-path-line-above-2'
+                                    : ''
+                                }`}
+                                style={{
+                                  backgroundColor: `${
```
Comment (allenan) [inline components/Txns/PocReceiptsV1.js:108]: does this need to be in an interpolated string?

File: components/Txns/PocReceiptsV1.js [hunk 4]
```diff
@@ -42,64 +42,151 @@ const PoCTableHeader = ({ tooltipText, title }) => {
...
+                          <div
+                            style={{
+                              display: 'flex',
+                              flexDirection: 'row',
+                              position: 'relative',
+                            }}
+                          >
+                            {idx !== 0 && (
+                              <span
+                                className={`poc-path-line-above ${
```
Comment (allenan) [inline components/Txns/PocReceiptsV1.js:101]: check out the `classnames` npm package. it's handy for conditional classes like this

File: pages/blocks/index.js [hunk 5]
```diff
@@ -23,7 +23,14 @@ function Blocks({ stats: initialStats, latestBlocks: initialLatestBlocks }) {
   const txnRate = meanBy(latestBlocks, 'transactionCount')
 
   return (
-    <AppLayout>
+    <AppLayout
+      title="Blocks"
+      description={
```
Comment (allenan) [inline pages/blocks/index.js:28]: the following 3 fields could also be written like `description="page description here"`

**Comment by danielcolinjames** [comment]
Updated in fresh PR: #123 to not merge in #107

**Pull request closed without merging**